importer/ThTh.py

```diff
@@ -0,0 +1,83 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class ThTh(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the Admin Location.
+
+        First, try the District (amphoe). These are never linked,
+        so match against external list.
+        If failed, use the Province ISO code matched against external
+        list.
+        """
+        adm_q = None
+        prov_dic = self.data_files["provinces"]
+        distr_dic = self.data_files["districts"]
+        iso = self.prov_iso
+        district = self.district
+
+        dist_match = utils.get_item_from_dict_by_key(dict_name=distr_dic,
+                                                     search_term=district,
+                                                     search_in="itemLabel")
+        if len(dist_match) == 1:
+            adm_q = dist_match[0]
```

Comment:
add to report already here. We care about a failed match even if we get a province later.